index.html

```diff
@@ -172,142 +172,273 @@ <h2 class="text-3xl font-extrabold text-gray-900 leading-9 sm:text-4xl sm:leadin
   </div>
 
   <script>
-    pageviewTiers =   ['10k', '100k', '200k', '500k', '1m', '2m', '5m', '10m', '10m+']
-    monthlyPriceTiers = ['9 / mo', '19 / mo', '29 / mo', '49 / mo', '69 / mo', '89 / mo', '129 / mo', '169 / mo', "<a class='underline' href='mailto:hello@plausible.io'>Contact us</a>"]
-    yearlyPriceTiers = ['90 / yr', '190 / yr', '290 / yr', '490 / yr', '690 / yr', '890 / yr', '1290 / yr', '1690 / yr', "<a class='underline' href='mailto:hello@plausible.io'>Contact us</a>"]
-  </script>
-  <div class="pb-16 mt-8 bg-white sm:mt-12 sm:pb-20 lg:pb-28" x-data="{volumeTier: 0, pricingYearly: false, currency: '$'}" x-init="fetch('/currency/us.json').then(response => response.json()).then(data => currency = data.currency)">
-    <div class="relative">
-      <div class="absolute inset-0 bg-gray-100 h-1/2"></div>
-      <div class="relative px-4 mx-auto max-w-screen-lg sm:px-6 lg:px-8">
-        <div class="max-w-lg mx-auto rounded-lg shadow-lg lg:max-w-none">
-          <div class="px-6 py-8 bg-white lg:p-12">
-            <div class="flex justify-between w-full">
-              <div>
-                <p class="text-sm font-medium tracking-wider text-gray-500 uppercase leading-6">
-                  Monthly pageviews
-                </p>
-                <div class="text-2xl font-extrabold leading-none text-gray-900" x-text="pageviewTiers[volumeTier]">
-                  10k
-                </div>
-              </div>
-              <div class="text-right">
-                <p class="text-sm font-medium tracking-wider text-gray-500 uppercase leading-6">
-                  Your price
-                </p>
-                <div class="text-2xl font-extrabold leading-none text-gray-900">
-                  <span x-show="volumeTier < 8" x-text="currency">$</span><!---
-                  --><span x-show="!pricingYearly" x-html="monthlyPriceTiers[volumeTier]">9 / mo</span><!--
-                  --><span x-show="pricingYearly" style="display: none;" x-html="yearlyPriceTiers[volumeTier]">90 / yr</span>
-                </div>
-              </div>
-            </div>
+    let currency = '$'
```

Comment:
This used to be in `x-data` which makes the variable reactive using Alpine.js. Meaning that if it is updated, the UI will update as well. By making it a regular javascript variable, it loses the reactive behaviour.

Try testing this locally by changing the contents of the `us.json` file. Make it `{currency: '€'}` instead. Now when the `fetch` request completes in the browser, the UI should updated with the € currency. With this code, I don't think it will.

Reply:
You're right! I just tried what you suggested and indeed - with the way I implemented this, only the first interaction with the slider will start displaying the new currency after the new request has been made. So it's better to keep it as it was - embedded in `x-data` and the request in `x-init`. Made that change in https://github.com/plausible/website/pull/476/commits/da37ebb03345ec524fb6de9eeeda62bc40e2a844